packages/dcos-launch/buildinfo.json

```diff
@@ -8,7 +8,7 @@
     "pyinstaller": {
       "kind": "url_extract",
       "url": "https://github.com/pyinstaller/pyinstaller/archive/8b491def32d02cedc912198930225cddfab2e871.zip",
-      "sha1": "d98d271f9b70021417b34a059766131a0d73fdb4"
+      "sha1": "643734c396f677fe56c5e67cdd69d18f58431126"
```

Comment:
See https://github.com/dcos/dcos/pull/1574/commits/cf4e951ba67b62c064ce088e223d8e3bab4671f5 -- this is necessary to get a build, apparently the upstream pyinstaller source broke for some reason. That commit is a backport of https://github.com/dcos/dcos/pull/1566/commits/773227c7e93329e810f28ad5c6927b8670f612c4.